Data Binding - PHP: Add dx-php-data Deprecation Updates (#8857)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chat/11 Data Binding.md

### dataSource

> For a more flexible solution than one achieved with the use of an array, specify the [dataSource](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource') property. For instance, with **dataSource**, Chat can handle server-side data processing through [Web API](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Web_API_and_MongoDB/), a [store](/concepts/70%20Data%20Binding/5%20Data%20Layer/1%20Creating%20DataSource/3%20What%20Are%20Stores.md '/Documentation/Guide/Data_Binding/Data_Layer/#Creating_DataSource/What_Are_Stores'), or a [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') object.

Reviewer: ```suggestion
For a more flexible solution than an array, use the [dataSource](/api-reference/10%20UI%20Components/dxChat/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource') property. For instance, **dataSource** allows Chat to handle server-side data processing through [Web API](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Web_API_and_MongoDB/), a [store](/concepts/70%20Data%20Binding/5%20Data%20Layer/1%20Creating%20DataSource/3%20What%20Are%20Stores.md '/Documentation/Guide/Data_Binding/Data_Layer/#Creating_DataSource/What_Are_Stores'), or a [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') object.
```
